zipline/utils/tradingcalendar.py

```diff
@@ -1,4 +1,3 @@
-#
```

Comment:
It should be `first time file is in edited in a year`. Though we, (myself included) haven't been following that 100%.